New packages: hunspell-pt_PT and hunspell-pt_PT-preao

lumigch opened the pull request:
Two new packages containing the dictionaries for pt_PT.  These are the pre- (hunspell-pt_PT-preao) and post- (hunspell-pt_PT) 1990 Orthographic Agreement.
Since these two packages I related, I decided to create a single pull-request.  This is my 1st pull-request for the project, so maybe you prefer two separate requests.  Please let me know any problems with these packages.

ericonr commented:
I believe it's ok in this case.

> These are the pre- (hunspell-pt_PT) and post- (hunspell-pt_PT-preao) 1990 Orthographic Agreement.

You switched them, right? Should be " post- (hunspell-pt_PT) and pre- (hunspell-pt_PT-preao) 1990 Orthographic Agreement.", I think, going by the package names.

Is the pre-acordo ortográfico one necessary? I know in Brazil we have basically switched over to post acordo by now.

sgn commented:
> Is the pre-acordo ortográfico one necessary? I know in Brazil we have basically switched over to post acordo by now.

I don't speak Portuguese but I could imagine it's useful for literature and linguistic researcher.
I don't have any strong opinion on including them. If someone find it's helpful, it's helpful.

ericonr commented on srcpkgs/hunspell-pt_PT-preao/template:
```diff
@@ -0,0 +1,18 @@
+revision=1
+wrksrc=${pkgname}-${version}
+depends="hunspell"
+short_desc="Portuguese dictionary for hunspell (prior 1990 Orthographic Agreement)"
```
prior to

ericonr commented on srcpkgs/hunspell-pt_PT-preao/template:
```diff
@@ -0,0 +1,18 @@
+checksum=9427f1b21b9a3e8126fce62d008e6215b0b631498232ae567859d363351fbac4
+
+do_install() {
+	vinstall ${wrksrc}/pt_PT-preao.aff 644 usr/share/hunspell
```
I don't think you should be using `${wrksrc}` here, templates are run inside the wrksrc by default.

ericonr commented on srcpkgs/hunspell-pt_PT-preao/template:
```diff
@@ -0,0 +1,18 @@
+pkgname=hunspell-pt_PT-preao
+version=20201029
+revision=1
+wrksrc=${pkgname}-${version}
```
This is default.

ericonr commented on srcpkgs/hunspell-pt_PT-preao/template:
```diff
@@ -0,0 +1,18 @@
+version=20201029
+revision=1
+wrksrc=${pkgname}-${version}
+depends="hunspell"
```
The other hunspell templates don't do this.

ericonr reviewed (commented):
Please apply the comments to both templates.

lumigch commented:
> I believe it's ok in this case.
> 
> > These are the pre- (hunspell-pt_PT) and post- (hunspell-pt_PT-preao) 1990 Orthographic Agreement.
> 
> You switched them, right? Should be " post- (hunspell-pt_PT) and pre- (hunspell-pt_PT-preao) 1990 Orthographic Agreement.", I think, going by the package names.

Ah!  Right, I did messed that up indeed.

> Is the pre-acordo ortográfico one necessary? I know in Brazil we have basically switched over to post acordo by now.

Well, in Portugal the pre-acordo is still wildly used -- there are newspapers, authors, etc that keep using it.  So, it is still relevant I believe.

lumigch commented:
> Please apply the comments to both templates.

Thanks a lot for the review.  I'll work through the comments later today and update my branch accordingly.

lumigch commented:
I've just pushed a new version of my branch, which includes all the suggestions made by @ericonr (thanks!).  Basically:
- dropped usage of 'wrksrc'
- dropped 'depends' on hunspell
- fixed typo on -preao package description

lumigch commented:
Looks like there are a bunch of CI failures that aren't really related to the hunspell packages.  The only relevant seems to be the hunspell-pt_PT-preao short description being 73 chars instead of the maximum allowed 72.  Is this a hard limit, or are the 73 chars allowed?  I can change "`Portuguese dictionary for hunspell (prior to 1990 Orthographic Agreement)`" to "`Portuguese dictionary for hunspell (prior to '90 Orthographic Agreement)`" if really needed.

ericonr commented:
CI is a bit wack recently.

Have you built the packages locally and tested them out?

lumigch commented:
> CI is a bit wack recently.
> 
> Have you built the packages locally and tested them out?

Yes, I did built them, installed and have been using them.

lumigch commented:
So... @ericonr sorry to bother you again but is there anything else I can/need to do to have these 2 packages merged into void-packages?

ericonr commented:
Don't worry, thanks for pinging.

Do you think the READMEs need to be added?

lumigch commented:
> Don't worry, thanks for pinging.
> 
> Do you think the READMEs need to be added?

I checked a few other distros packages and some include it, some don't.

The most relevant argument to include it is that the [mozilla addon](https://addons.mozilla.org/en-US/firefox/addon/european-portuguese-spellcheck/), which seems to be packaged by the dictionaries authors themselves, **does** include the README file.

Anyway, I'm OK either way: I can re-push the branch without it if you rather not include it.

ericonr commented:
Sounds reasonable to me. I will wait to merge it, because the builder is currently building libreoffice and that takes a while; merging new packages in the meantime can lead to it not being built.
